Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -574,13 +632,31 @@ public void OnLoggedOut()
             userLoggedIn = false;
         }
 
+        /// <summary>
+        /// Unsubscribes from invitations and Peer2Peer message requests.
+        /// This should be called before the application finishes shutting down in order to properly remove Native notifications.
+        /// TODO: This function has no references, and is incomplete in its implementation. This will likely be removed.
+        /// </summary>
+        [Obsolete]
         private void OnShutDown()
         {
             DestroyAllSessions();
             UnsubscribeFromGameInvites();
             UnsubscribeToSessionMessageConnectionRequests();
         }
 
+        /// <summary>
+        /// Informs the <see cref="UIInterface"/> that a UI Event has been processed.
+        /// Some functions in the EOS SDK return a UIEventId. When this happens, it is required that this function be called to released by the SDK.
+        /// For example, if you are subscribed to <see cref="OnJoinSessionAcceptedListener(ref JoinSessionAcceptedCallbackInfo)"/>, after a successful call to <see cref="SessionsInterface.JoinSession(ref JoinSessionOptions, object, OnJoinSessionCallback)"/>,
+        /// the notification function will receive a <see cref="JoinSessionAcceptedCallbackInfo.UiEventId"/>.
+        /// You can use this event id to create a <see cref="SessionDetails"/> object by calling <see cref="MakeSessionHandleByEventId(ulong)"/>.
+        /// After you've done so, or if you aren't going to use this handle, you must call this function to release the handle in the EOS SDK.
+        /// Most of the functions in the EOS SDK that require this will mention it in the comments of the callback function. Anything with a UiEventId should have this called.
+        /// 
+        /// TODO: This is hard coded to only handle the <see cref="JoinUiEvent"/> case. It must be changed to also accept the event's id in order to function properly.
```

Comment:
nit: Another example where the contraction can be removed without altering the meaning of the comment.

Reply:
This contraction got eliminated with the rephrase, but I'll note to use "you have" if this ever comes up again.